nuxeo-services/nuxeo-platform-dublincore/src/main/java/org/nuxeo/ecm/platform/dublincore/constants/DublinCoreConstants.java

```diff
@@ -32,28 +33,72 @@ private DublinCoreConstants() {
 
     public static final String DUBLINCORE_SCHEMA_NAME = "dublincore";
 
+    public static final String DUBLINCORE_CONTRIBUTORS_PROPERTY = "dc:contributors";
+
+    public static final String DUBLINCORE_CREATED_DATE_PROPERTY = "dc:created";
+
     public static final String DUBLINCORE_CREATOR_PROPERTY = "dc:creator";
 
+    public static final String DUBLINCORE_DESCRIPTION_PROPERTY = "dc:description";
+
     public static final String DUBLINCORE_ISSUED_DATE_PROPERTY = "dc:issued";
 
+    public static final String DUBLINCORE_LAST_CONTRIBUTOR_PROPERTY = "dc:lastContributor";
+
     public static final String DUBLINCORE_MODIFIED_DATE_PROPERTY = "dc:modified";
 
-    public static final String DUBLINCORE_CREATED_DATE_PROPERTY = "dc:created";
+    public static final String DUBLINCORE_COVERAGE_PROPERTY = "dc:coverage";
 
-    public static final String DUBLINCORE_CONTRIBUTORS_PROPERTY = "dc:contributors";
+    public static final String DUBLINCORE_VALID_PROPERTY = "dc:valid";
 
-    public static final String DUBLINCORE_LAST_CONTRIBUTOR_PROPERTY = "dc:lastContributor";
+    public static final String DUBLINCORE_RIGHTS_PROPERTY = "dc:rights";
+
+    public static final String DUBLINCORE_EXPIRED_PROPERTY = "dc:expired";
+
+    public static final String DUBLINCORE_FORMAT_PROPERTY = "dc:format";
+
+    public static final String DUBLINCORE_TITLE_PROPERTY = "dc:title";
+
+    public static final String DUBLINCORE_NATURE_PROPERTY = "dc:nature";
+
+    public static final String DUBLINCORE_SUBJECTS_PROPERTY = "dc:subjects";
+
+    public static final String DUBLINCORE_SOURCE_PROPERTY = "dc:source";
+
+    public static final String DUBLINCORE_PUBLISHER_PROPERTY = "dc:publisher";
+
+    public static final String DUBLINCORE_CONTRIBUTORS = "contributors";
+
+    public static final String DUBLINCORE_CREATED_DATE = "created";
 
     public static final String DUBLINCORE_CREATOR = "creator";
 
+    public static final String DUBLINCORE_DESCRIPTION = "description";
```

Comment:
Usage of the "bare" properties (without prefix) is really not a good idea, people should always use full property names like "dc:description". So I'd rather not have these new constants for things we don't want in the code.

Reply:
I agree with it, just tried to be consistent with the existing

Reply:
In some cases (but no idea about the reason), you could use doc.getProperty(DUBLINCORE_SCHEMA_NAME, DUBLINCORE_CREATOR)

Reply:
Yes that's the "old style". We're trying to discourage its usage. Although there hasn't been a task force to clean up our code yet.

Reply:
ok done

Reply:
It was added for coherency purposes with the existing ones.
Should we remove them or just mark as @deprecated, @efge?

Reply:
If we deprecate them we should first cleanup our own codebase, so I think it's a separate (but related) issue.